src/components/CreateResetPassword/components/ChangePasswordConfirmation/ChangePasswordConfirmation.css

```diff
@@ -0,0 +1,23 @@
+@import "@folio/stripes-components/lib/variables.css";
+
+.wrap {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+}
+
+.centered {
+  position: relative;
+  max-width: 500px;
+  width: 100%;
+  padding: 30px 15px;
+  text-align: center;
+}
+
+button.submitButton {
```

Comment:
Can we avoid the `button` tag in selector?

Reply:
Great idea, thanks